app/Dockerfile

```diff
@@ -5,13 +5,14 @@ COPY package*.json /app/
 FROM base AS builder
 RUN npm install
 COPY . .
+RUN npm run build
```

Comment:
This will result in additional build time for the dev image, do we want that?

Reply:
Frontend image only gets built once for devs. If I run that command as prod I would also need to run npm install to get all dependencies. By building it at the build step I can keep the prod image very small by skipping the installation of dependencies. And since the frontend image does not get built that often I find that the added amount of time it takes to build the frontend image is well worth the space we'll be saving in the prod image size